app/visits/edit/controller.js

```diff
@@ -516,6 +522,22 @@ export default AbstractEditController.extend(AddNewPatient, ChargeActions, Diagn
       this.send('update', true);
     },
 
+    printReport(report) {
```

Comment:
I'm not too keen on how this was implemented.  I think a better approach would be when clicking on the Print button, you are directed to the report page and print automatically fires from that page.  You can use the {{print-this}} component to automatically  print the page on render, so if you pass in a queryParam, in the template you can add the following to auto print:
```hbs
{{#if printQueryParam}} 
  {{print-this}}
{{/if}}
```